mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
if you plan on using mypy, you should add a pre-commit linter for it.  https://github.com/pre-commit/mirrors-mypy